.travis.yml

```diff
@@ -82,8 +90,8 @@ script:
   - if [[ $TRAVIS_PULL_REQUEST ]]; then
       export NOSEATTR="!nonpublic";
     fi
-  - nosetests indra -v --with-doctest --with-doctest-ignore-unicode
-        --with-coverage --cover-inclusive --cover-package=indra -a $NOSEATTR
+  - nosetests -v -s --process-restartworker indra/tests/test_tees.py -a $NOSEATTR
```

Comment:
Would this work even if we took out the `-s` option? If yes, then let's get rid of that, it creates way too much logging output!